packages/eslint-config-salesforce-typescript/package.json

```diff
@@ -22,8 +22,8 @@
     "url": "https://github.com/forcedotcom/sfdx-dev-packages/issues"
   },
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^2.30.0",
-    "@typescript-eslint/parser": "^2.30.0",
+    "@typescript-eslint/eslint-plugin": "^4.2.0",
+    "@typescript-eslint/parser": "^4.2.0",
     "eslint": "^6.8.0",
```

Comment:
Geesh, I wonder if we also need a similar story around keeping eslint up-to-date... it is never ending.